lightning/lightning-client.md

```diff
@@ -192,7 +193,7 @@ $ lnd --version
 **Example** of expected output:
```

Comment:
This is only an *Example* of expected output, "Example" is only an example, it wouldn't have to be the same